modules/teos.nix

```diff
@@ -0,0 +1,178 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  options.services.teos = {
+    enable = mkEnableOption "Lightning watchtower compliant with BOLT13, written in Rust";
+    address = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = mdDoc "Address to listen for RPC connections.";
+    };
+    port = mkOption {
+      type = types.port;
+      default = 8814;
+      description = mdDoc "Port to listen for RPC connections.";
+    };
+    externalPort = mkOption {
+      type = types.port;
+      default = 2121;
+      description = mdDoc "Override an external port to listen for onion API connections.";
+    };
```

Comment:
this should not be here but you should setup an externalPort in modules/onion-services.nix, at the end of the file something like: 
```
nix-bitcoin.onionServices.teos = {
  enable = true;
  externalPort = 2121;
};
```

Reply:
if you don't want to touch this module, you can set it up like in my comment for the example...